handlers/create.go

```diff
@@ -216,89 +219,42 @@ func getBaseMachineDir(uuid string) (string, error) {
-			cmd = append(cmd, "--digitalocean-image", machine.DigitaloceanConfig.Image)
-		}
-		if machine.DigitaloceanConfig.Size != "" {
-			cmd = append(cmd, "--digitalocean-size", machine.DigitaloceanConfig.Size)
-		}
-		if machine.DigitaloceanConfig.Region != "" {
-			cmd = append(cmd, "--digitalocean-region", machine.DigitaloceanConfig.Region)
-		}
-		if machine.DigitaloceanConfig.AccessToken != "" {
-			cmd = append(cmd, "--digitalocean-access-token", machine.DigitaloceanConfig.AccessToken)
-		}
-		if machine.DigitaloceanConfig.Ipv6 {
-			cmd = append(cmd, "--digitalocean-ipv6")
-		}
-		if machine.DigitaloceanConfig.PrivateNetworking {
-			cmd = append(cmd, "--digitalocean-private-networking")
-		}
-		if machine.DigitaloceanConfig.Backups {
-			cmd = append(cmd, "--digitalocean-backups")
-		}
-	case "amazonec2":
-		cmd = append(cmd, "amazonec2")
-		if machine.Amazonec2Config.AccessKey != "" {
-			cmd = append(cmd, "--amazonec2-access-key", machine.Amazonec2Config.AccessKey)
-		}
-		if machine.Amazonec2Config.SecretKey != "" {
-			cmd = append(cmd, "--amazonec2-secret-key", machine.Amazonec2Config.SecretKey)
-		}
-		if machine.Amazonec2Config.VpcId != "" {
-			cmd = append(cmd, "--amazonec2-vpc-id", machine.Amazonec2Config.VpcId)
-		}
-		if machine.Amazonec2Config.Ami != "" {
-			cmd = append(cmd, "--amazonec2-ami", machine.Amazonec2Config.Ami)
-		}
-		if machine.Amazonec2Config.SessionToken != "" {
-			cmd = append(cmd, "--amazonec2-session-token", machine.Amazonec2Config.SessionToken)
-		}
-		if machine.Amazonec2Config.Region != "" {
-			cmd = append(cmd, "--amazonec2-region", machine.Amazonec2Config.Region)
-		}
-		if machine.Amazonec2Config.Zone != "" {
-			cmd = append(cmd, "--amazonec2-zone", machine.Amazonec2Config.Zone)
-		}
-		if machine.Amazonec2Config.SubnetId != "" {
-			cmd = append(cmd, "--amazonec2-subnet-id", machine.Amazonec2Config.SubnetId)
-		}
-		if machine.Amazonec2Config.SecurityGroup != "" {
-			cmd = append(cmd, "--amazonec2-security-group", machine.Amazonec2Config.SecurityGroup)
-		}
-		if machine.Amazonec2Config.InstanceType != "" {
-			cmd = append(cmd, "--amazonec2-instance-type", machine.Amazonec2Config.InstanceType)
-		}
-		if machine.Amazonec2Config.RootSize != "" {
-			cmd = append(cmd, "--amazonec2-root-size", machine.Amazonec2Config.RootSize)
-		}
-		if machine.Amazonec2Config.IamInstanceProfile != "" {
-			cmd = append(cmd, "--amazonec2-iam-instance-profile", machine.Amazonec2Config.AccessKey)
-		}
-	case "virtualbox":
-		cmd = append(cmd, "virtualbox")
-		if machine.VirtualboxConfig.Boot2dockerUrl != "" {
-			cmd = append(cmd, "--virtualbox-boot2docker-url", machine.VirtualboxConfig.Boot2dockerUrl)
-		}
-		if machine.VirtualboxConfig.DiskSize != "" {
-			cmd = append(cmd, "--virtualbox-disk-size", machine.VirtualboxConfig.DiskSize)
-		}
-		if machine.VirtualboxConfig.Memory != "" {
-			cmd = append(cmd, "--virtualbox-memory", machine.VirtualboxConfig.Memory)
+	sDriver := strings.ToLower(machine.Driver)
```

Comment:
We should avoid hungarian notation. Can you find better names for `sDriver, rConfig, tConfig, and rMachine`?